internal/handlers/user_response_handler.go

```diff
@@ -0,0 +1,437 @@
+package handlers
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/antonioo83/license-server/config"
+	"github.com/antonioo83/license-server/internal/models"
+	"github.com/antonioo83/license-server/internal/repositories/interfaces"
+	"github.com/antonioo83/license-server/internal/utils"
+	"github.com/go-playground/validator/v10"
+	"github.com/google/uuid"
+	"net/http"
+	"strconv"
+	"strings"
+)
+
+type UserRouteParameters struct {
+	Config               config.Config
+	UserRepository       interfaces.UserRepository
+	ActionRepository     interfaces.UserActionRepository
+	PermissionRepository interfaces.UserPermissionRepository
+}
+
+func GetCreatedUserResponse(r *http.Request, w http.ResponseWriter, param UserRouteParameters) {
```

Comment:
param  у тебя статичный. лучше тогда сделать из UserRouteParameters UserRoute 
```suggestion
func (param UserRoute)GetCreatedUserResponse(r *http.Request, w http.ResponseWriter) {
```
так выходит логичней

Reply:
в принципе таким образом касается всех роутов :)